resources/application.yaml

```diff
@@ -1,94 +1,29 @@
+# Documentation about this file can be found in this file : /docs/install/1-CONFIG.md
+
 security:
   disabled: true
   jwt:
     key: super-secret
-  login:
-    admin:
-      pwd: admin
-  external_auth:
-    url: ""
-    default_group_role: 10
-#    group_mapping:
-#      id_ext: id_int
-#      ...
-    add_ext_groups: false
-
-
 
 db:
   url: "sqlite:///database.db"
-  #pool_recycle:
 
 storage:
-  tmp_dir: /tmp
+  tmp_dir: ./tmp
   matrixstore: ./matrices
-  archive_dir: examples/archives
-  allow_deletion: false # indicate if studies found in non default workspace can be deleted by the application
-  #matrix_gc_sleeping_time: 3600 # time in seconds to sleep between two garbage collection
-  #matrix_gc_dry_run: False # Skip matrix effective deletion
-  #auto_archive_sleeping_time: 3600 # time in seconds to sleep between two auto archival checks
-  #auto_archive_dry_run: True # Skip auto archive effective archival
-  #auto_archive_threshold_days: 60 # number of days after last study access when the study should be archived
-  #auto_archive_max_parallel: 5 # max auto archival tasks in parallel
+  archive_dir: ./examples/archives
   workspaces:
-    default: # required, no filters applied, this folder is not watched
-      path: examples/internal_studies/
-    # other workspaces can be added
-    # if a directory is to be ignored by the watcher, place a file named AW_NO_SCAN inside
-    tmp:
-      path: examples/studies/
-      # filter_in: ['.*'] # default to '.*'
-      # filter_out: [] # default to empty
-      # groups: [] # default empty
+    default: 
+      path: ./examples/internal_studies/
+    studies:
+      path: ./examples/studies/
 
 launcher:
-  default: local
   local:
     binaries:
-      700: path/to/700
-#  slurm:
-#    local_workspace: path/to/workspace
-#    username: username
-#    hostname: 0.0.0.0
-#    port: 22
-#    private_key_file: path/to/key
-#    key_password: key_password
-#    password: password_is_optional_but_necessary_if_key_is_absent
-#    default_wait_time: 900
-#    default_time_limit: 172800
-#    default_n_cpu: 12
-#    default_json_db_name: launcher_db.json
-#    slurm_script_path: /path/to/launchantares_v1.1.3.sh
-#    db_primary_key: name
-#    antares_versions_on_remote_server :
-#      - "610"
-#      - "700"
-#      - "710"
-#      - "720"
-#      - "800"
-
+      850: ./AntaresWeb/antares_solver/antares-8.5-solver
```

Comment:
The path to the solver binary is doubly incorrect because this directory doesn't exist, and the version number is outdated.

Stepping back a bit, the real issue is understanding the purpose of the `application.yaml` file. From my perspective, this file is merely an example that developers can use to create their own configuration files for integration testing in a development environment. If this is indeed the intended use case, a better solution would be to relocate this file to the documentation, for instance, in a directory like `docs/assets/examples`, and include it as an example at the end of the description in `docs/install/1-CONFIG.md`.